app/assets/stylesheets/govuk-component/_tasklist.scss

```diff
@@ -0,0 +1,358 @@
+  width: $number-circle-size;
+  height: $number-circle-size;
+  margin-top: -($number-circle-size / 2);
+  margin-left: -($number-circle-size / 2);
+  color: $black;
+  background: $white;
+  border: solid 3px $grey-2;
+  border-radius: 100px;
+  text-align: center;
+
+  .pub-c-tasklist--large & {
+    @include media(tablet) {
+      @include bold-24($number-circle-size * 0.9);
+    }
+  }
+}
+
+.pub-c-tasklist__step {
+  position: relative;
+  list-style: none;
+  border-top: 1px solid $grey-2;
+
+  // tasklist vertical line
+  &:before {
+    @include core-19;
+
+    content: "";
+    position: absolute;
+    z-index: 1;
+    width: $line-width;
+    height: 100%;
+    left: 0;
+    top: 0;
+    margin-left: -($line-width / 2);
+    background: $grey-2;
+  }
+
+  // 'dash' at end of tasklist vertical line
+  &:last-child {
+    &:after {
+      @include core-19;
+
+      content: "";
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      width: $number-circle-size / 2;
+      height: $line-width;
+      margin-left: -($number-circle-size / 4);
+      background: $grey-2;
+    }
+  }
+
+  .pub-c-tasklist--large & {
+    @include media(tablet) {
+      &:before {
+        @include core-24;
+      }
+
+      &:last-child:after {
+        @include core-24;
+
+        width: $number-circle-size / 2.5;
+        margin-left: -($number-circle-size / 5);
+      }
+    }
+  }
+}
+
+.pub-c-tasklist__step--active {
+  .pub-c-tasklist__number {
+    border-color: $black;
+  }
+
+  &:before,
+  &:last-child:after {
+    background: $black;
+  }
+}
+
+.pub-c-tasklist__section {
+  &:after {
+    content: "";
+    display: block;
+    margin-left: $first-indent-small;
+    border-bottom: 1px solid $border-colour;
+  }
+
+  &:last-child:after {
+    border-bottom: 0;
+  }
+
+  .pub-c-tasklist--large & {
+    &:after {
+      margin-left: $first-indent;
+    }
+  }
+}
+
+.pub-c-tasklist__header {
+  position: relative;
+  padding-top: 0.9em;
+  padding-bottom: 0.9em;
+
+  .pub-c-tasklist--active & {
+    cursor: pointer;
+
+    &:hover {
+      background: $grey-3;
+    }
+  }
+}
+
+// having two icons makes overriding the styles for the small tasklist simpler
+
+$icon-size: 10px;
+$icon-size-large: 12px;
+
+.pub-c-tasklist__icon {
+  position: absolute;
+  top: 50%;
+  left: $first-indent-small;
+  height: $icon-size;
+  width: $icon-size;
+  margin-top: -($icon-size / 2) - 2;
+  background-image: image-url('icon-plus-minus-small.png'); // PNG fallback for SVG
+  background: image-url("icon-plus-minus.svg"), linear-gradient(transparent, transparent); // http://pauginer.com/post/36614680636/invisible-gradient-technique
+  background-repeat: no-repeat;
+
+  .pub-c-tasklist--large & {
+    @include media(tablet) {
+      left: $first-indent;
+      height: $icon-size-large;
+      width: $icon-size-large;
+      margin-top: -($icon-size-large / 2) - 2;
+      // repeating these two lines for the benefit of IE8, which doesn't support background size
```

Comment:
Could you indicate why this needs repeating?

Reply:
Will clarify.